markdown/ch8.markdown

```diff
@@ -1,6 +1,18 @@
 # Big Data in Little Laptop with Toolz
 
-Whenever I think too hard about streaming data analysis, my head hurts.
+> GRACIE: A knife? The guy's twelve feet tall!
+> JACK: Seven. Hey, don't worry, I think I can handle him.
+>
+> — Jack Burton, Big Trouble in Little China
+
+Although streaming is not really a SciPy feature, it is a way of writing code
```

Comment:
The "that is critical to efficiently processing" part doesn't parse well for me.